### PR TITLE
Combained back with front for signIn functionality

### PR DESCRIPTION
**Pull request by TarasOrlovskyi**
1. Added thymeleaf into signIn.html
2. Added: method Get, @override into SignInServlet & changed doPost
   there
3. Added message into Context in PageGenerator

### CODE REVIEW

**Review by Greengrer — DISMISSED** [review]

File: src/main/java/com/vinylteam/vinyl/web/servlets/SignInServlet.java [hunk 1]
```diff
@@ -36,11 +46,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         } else if (checkResult == SignInCheckResult.OK_NOT_VERIFIED) {
             response.setStatus(HttpServletResponse.SC_SEE_OTHER);
             logger.debug("Set response status to {'status':{}}", HttpServletResponse.SC_SEE_OTHER);
-            response.sendRedirect(notVerifiedRedirect);
+            attributes.put("verified", "Sorry, your email has not been verified. Please go to your mailbox and follow the link to confirm your registration.");
```
Comment (AlexKievDeveloper) [inline src/main/java/com/vinylteam/vinyl/web/servlets/SignInServlet.java]: Maybe more comfortable use word "message" as a key instead "verified"?

File: src/main/java/com/vinylteam/vinyl/web/servlets/SignInServlet.java [hunk 2]
```diff
@@ -36,11 +46,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+            PageGenerator.getInstance().process("signIn", attributes, response.getWriter());
         } else if (checkResult == SignInCheckResult.FAIL) {
             response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
             logger.debug("Set response status to {'status':{}}", HttpServletResponse.SC_BAD_REQUEST);
-            response.sendRedirect(failRedirect);
+            attributes.put("incorrectCredentials", "Sorry, login or password is not correct, please check yours credentials and try again.");
```
Comment (AlexKievDeveloper) [inline src/main/java/com/vinylteam/vinyl/web/servlets/SignInServlet.java]: maybe more comfortable use word "message" as a key instead "incorrectCredentials"?

**Review by Greengrer — DISMISSED** [review]
*Combined(as combination)

File: src/main/resources/templates/signIn.html [hunk 3]
```diff
@@ -73,15 +74,20 @@
     <!-- content of main -->
 
     <div class="container">
-      <div class="content-sign-in content">
+      <div class="content-sign-in">
+
+        <div class="user__message">
+          <div th:if="${verified != null}" th:text="${verified}"></div>
+          <div th:if="${incorrectCredentials != null}" th:text="${incorrectCredentials}"></div>
```
Comment (AlexKievDeveloper) [inline src/main/resources/templates/signIn.html]: If we will use word "message" as a key, you can leave only one check for condition

**Review by AlexKievDeveloper — CHANGES_REQUESTED** [review]

**Review by Greengrer — APPROVED** [review]

**Review by AlexKievDeveloper — APPROVED** [review]

**Pull request merged**